package/MDAnalysis/core/groups.py

```diff
@@ -1674,6 +1674,64 @@ def unwrap(self, compound='fragments', reference='com', inplace=True):
             positions = positions[atoms._unique_restore_mask]
         return positions
 
+    def arrange_closest(self, target_position, reference='com', inplace=True):
```

Comment:
I see some use cases where this should work for multiple target positions, i.e., an array of target coordinates. (Think of the micelles example)
Furthermore, the method should have a `compound` parameter so that the shifts can be performed per compound.